### DISCUSSION TITLE
Hide songs

### DISCUSSION THREAD

**Discussion by drybin24** [fast4x/RiMusic#1597] [Ideas]
I really don't see a point in "hide" button in current version of RiMusic. Don't you think that it would be better if such songs (that marked as hidden) were always skipped and never played again? 

I personally hide every song on Spotify that didn't arouse some good emotions inside of me during the FIRST listening. I mean, about 90% of songs_I_listen are marked as hidden. (I suppose it's normal for other people 😀)

Unfortunately, there are a lot of the **same** songs that are in many **different** playlists. And when I want to listen to another playlist in Spotify, I see that there are already several songs marked as hidden, which will be skipped: I won't listen the songs that I didn't like a week ago, for example.

I hope you'll understand my issue and solve it as soon as you can.

Best wishes

**Comment by drybin24** [comment]
Also, I think it will be great, if you make an automatic "hiding" of songs that a user listened to more than 2 times and he/she didn't press a like-button. It could be an optional feature (I just suppose that there are such people who can press a like-button, even after they had already listened to the same song several times)

**Comment by fast4x** [comment]
Hi, add a ticket for this feature request

**Comment by fast4x** [comment]
Hided songs are now marked in all place included playlists, so it can be distingued from other songs.
